app/profiles/queries/searchProfiles.ts

```diff
@@ -12,7 +12,7 @@ export default resolver.pipe(resolver.authorize(), async (search: String) => {
   `
   let result
   if (search && search !== "") {
-    const prefixSearch = search + "*"
+    const prefixSearch = "\"" + search + "\"*"
```

Comment:
Allow searching profiles by email, by quoting (") the search string, thus allowing punctuation characters like . and @ that were causing errors.